my_project/dj_database_url.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why you copying dj_database_url instead of `pip install dj_database_url` and import it into `settings.py`?